src/command_modules/azure-cli-role/setup.py

```diff
@@ -34,7 +34,7 @@
     'azure-cli-core',
     'azure-mgmt-authorization==0.30.0rc6',
     'azure-graphrbac==0.30.0rc6',
-    'azure-keyvault==0.3.0',
+    'azure-keyvault==0.3.4',
```

Comment:
Changing the API versions of other command modules may break their tests depending on whether there were breaking changes between the different SDK versions.